sharepoint/sharepoint-ps/sharepoint-online/Invoke-SPOSiteSwap.md

```diff
@@ -166,7 +169,7 @@ Accept wildcard characters: False
 
 Overrides and ignores any warnings that have been identified by the Page Diagnostic Tool that are preventing a swap from being initiated.
 
-For example, if you receive the warning "Invoke-SPOSiteSwap : Errors: 0; Warnings: 1; Details: The site is not a communications site or it has the classic publishing feature enabled." then this warning can be overriden and ignored using this parameter.
+For example, if you receive the warning "Invoke-SPOSiteSwap : Errors: 0; Warnings: 1; Details: 2 items are not in CDN, please enable CDN." Warnings can be overriden using this parameter but the items should be reviewed and resolved for long-term stability and performance.
```

Comment:
Is the original warning message not valid? Has it been replaced by the new message?

Reply:
Yes, a classic site is not supported for site swap and should never have been used in the documentation. We only support swapping a modern site to the root. If you look at the site swap documentation you will see that it has been updated as well